src/v2/baseCoin.js

```diff
@@ -109,6 +110,23 @@ class BaseCoin {
     });
   }
 
+
+  static setupOffchainTokens(coins, bitgo) {
+    if (!OFCToken) {
+      OFCToken = require('./coins/ofcToken');
+    }
+
+    bitgo.fetchConstants()
+    .then(function(constants) {
+      const tokens = constants.ofc.tokens;
```

Comment:
`_.get(constants, 'ofc.tokens')` so we don't blow up on the off chance that the ofc property is missing from the constants object.